src/include/schema.h

```diff
@@ -111,6 +111,11 @@ struct __wt_import_list {
 /*
  * WT_WITH_LOCK_WAIT --
  *	Wait for a lock, perform an operation, drop the lock.
+ *  This macro lets us treat spinlocks as re-entrant. The session is only accessible by its owning
+ *  thread so the sessions lock flags can be considered as thread-local tracking for whether we're
```

Comment:
Minor nit: "... so the sessions lock flags..." `sessions` should be `session's`

Reply:
Thanks, fixed in [a03f13c](https://github.com/wiredtiger/wiredtiger/pull/11461/commits/a03f13cf2f6d200cfc5488dd0b41555ee3da998f)